src/utilities/CMakeLists.txt

```diff
@@ -125,6 +128,24 @@ set(filetypes_src
   filetypes/WorkflowStepResult.cpp
 )
 
+set(alfalfa_src
```

Comment:
Perhaps src/utilities/alfalfa should come up a level as utilities is taking on a lot. Perhaps src/alfalfa?

Reply:
That's what i originally had, but then i got humble. I'm happy to move it back up to that level again. It would also make some of the ruby bindings a bit easier.

Reply:
done